functions/src/models/healthSummaryData.ts

```diff
@@ -7,27 +7,162 @@
+      : null
+  }
+
+  // Computed Properties - Symptom Scores
+
+  get latestSymptomScore(): SymptomScore | null {
+    return this.symptomScores.at(0) ?? null
+  }
+
+  get secondLatestSymptomScore(): SymptomScore | null {
+    return this.symptomScores.at(1) ?? null
+  }
+
+  // Computed Properties - KeyPoints
+
+  get dizzinessCategory(): HealthSummaryDizzinessCategory | null {
+    const latestScore = this.latestSymptomScore?.dizzinessScore ?? null
+    const secondLatestScore =
+      this.secondLatestSymptomScore?.dizzinessScore ?? null
+
+    if (latestScore === null || secondLatestScore === null) return null
+
+    return latestScore - secondLatestScore < 0 ?
+        HealthSummaryDizzinessCategory.WORSENING
+      : HealthSummaryDizzinessCategory.STABLE_OR_IMPROVING
+  }
+
+  get recommendationsCategory(): HealthSummaryMedicationRecommendationsCategory | null {
+    const hasOptimizations = this.recommendations.some((recommendation) =>
+      [
+        UserMedicationRecommendationType.improvementAvailable,
+        UserMedicationRecommendationType.notStarted,
+      ].includes(recommendation.displayInformation.type),
+    )
+    if (hasOptimizations)
+      return HealthSummaryMedicationRecommendationsCategory.OPTIMIZATIONS_AVAILABLE
+
+    const hasObservationsRequired = this.recommendations.some(
+      (recommendation) =>
+        [
+          UserMedicationRecommendationType.moreLabObservationsRequired,
+          UserMedicationRecommendationType.morePatientObservationsRequired,
+        ].includes(recommendation.displayInformation.type),
+    )
+    if (hasObservationsRequired)
+      return HealthSummaryMedicationRecommendationsCategory.OBSERVATIONS_REQUIRED
+
+    const hasAtTarget = this.recommendations.some((recommendation) =>
+      [
+        UserMedicationRecommendationType.personalTargetDoseReached,
+        UserMedicationRecommendationType.targetDoseReached,
+      ].includes(recommendation.displayInformation.type),
+    )
+    return hasAtTarget ?
+        HealthSummaryMedicationRecommendationsCategory.AT_TARGET
+      : null
+  }
+
+  get symptomScoreCategory(): HealthSummarySymptomScoreCategory | null {
+    const latestScore = this.latestSymptomScore
+    const secondLatestScore = this.secondLatestSymptomScore
+
+    if (latestScore === null || secondLatestScore === null) return null
+
+    if (latestScore.overallScore - secondLatestScore.overallScore <= -10)
+      return HealthSummarySymptomScoreCategory.WORSENING
+
+    if (latestScore.overallScore < 90)
+      return HealthSummarySymptomScoreCategory.LOW_STABLE_OR_IMPROVING
+
+    return HealthSummarySymptomScoreCategory.HIGH_STABLE_OR_IMPROVING
+  }
+
+  get weightCategory(): HealthSummaryWeightCategory {
+    const averageWeight = this.averageBodyWeight
+    const latestWeight = this.latestBodyWeight
+    if (averageWeight === null || latestWeight === null)
+      return HealthSummaryWeightCategory.MISSING
+
+    return latestWeight - averageWeight > 1 ?
```

Comment:
Not sure if I missed this here, but shouldn't this be 3lb larger to be counted as an increase?

Reply:
Yes, you are right - now that we have this confirmed to always be a check for a 3lbs increase, I will also adapt the other case

Reply:
should be fixed now